std/range/package.d

```diff
@@ -9701,8 +9701,11 @@ Returns $(D true) if and only if $(D value) can be found in $(D
 range), which is assumed to be sorted. Performs $(BIGOH log(r.length))
 evaluations of $(D pred). See also STL's $(HTTP
 sgi.com/tech/stl/binary_search.html, binary_search).
- */
 
+$(RED Deprecated - please use $(REF canFind, std,algorithm.searching) instead).
+ */
+    // @@@DEPRECATED_2018-02@@@
+    deprecated("Please use std.algorithm.searching : canFind instead.")
```

Comment:
@jmdavis - I hope I have done it correctly this time?
IIRC the stages were:
- mark in documentation as deprecated
- remove from documentation
- remove entirely